GdscBackend.Tests/TeamsControllerTests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections.Generic;
+using System.Linq;
+using FactoryBot;
+using GdscBackend.Controllers.v1;
+using GdscBackend.Database;
+using GdscBackend.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GdscBackend.Tests
+{
+    public class TeamsControllerTests : TestingBase
+    {
+        private readonly IEnumerable<TeamModel> _testData = _getTestData();
+        
+        public TeamsControllerTests(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [Fact]
+        public async void Delete()
```

Comment:
Test methods should explicitly tell what they do and what is expected. Ex: `Delete_Returns_OkResult`, `Delete_Returns_Error_When_IdNotFoundInRepo`. Please rename your method accordingly